app/views/games/edit.html.erb

```diff
@@ -1,8 +1,34 @@
-<h1>Editing game</h1>
+<h1 style="text-align: center">Game <%= @game.id.split("-").first %></h1>
```

Comment:
This looks too similar with the #show. I think we can make the name `Editing Game <%= @game.id.split("-").first %>` so it's a bit more clear what we do 🤔